cmd/bucket-replication.go

```diff
@@ -769,6 +769,20 @@ func putReplicationOpts(ctx context.Context, sc string, objInfo ObjectInfo) (put
 		// In case of SSE-C objects copy the allowed internal headers as well
 		if !crypto.SSEC.IsEncrypted(objInfo.UserDefined) || !slices.Contains(maps.Keys(validSSEReplicationHeaders), k) {
 			if stringsHasPrefixFold(k, ReservedMetadataPrefixLower) {
+				if strings.EqualFold(k, ReservedMetadataPrefixLower+"crc") {
+					cs := objInfo.decryptChecksums(0)
+					if cs != nil {
+						for k, v := range cs {
+							cksum := hash.NewChecksumString(k, v)
+							if cksum == nil {
+								continue
+							}
+							if cksum.Valid() {
+								meta[cksum.Type.Key()] = v
+							}
+						}
+					}
+				}
```

Comment:
this is not the only checksum @poornas we have support for sha1, sha256 and crc32c?

Reply:
https://github.com/minio/minio/blob/master/internal/hash/checksum.go#L91 - all of it is covered

Reply:
you raise interesting questions - do not know how SSEC / multipark checksums need to be handled. will do some testing , converting this to draft PR for now.